Marlin/src/inc/SanityCheck.h

```diff
@@ -1046,6 +1047,27 @@ static_assert(Y_MAX_LENGTH >= Y_BED_SIZE, "Movement bounds (Y_MIN_POS, Y_MAX_POS
     #error "RACK_AND_PINION_PROBE requires Z_PROBE_DEPLOY_X and Z_PROBE_RETRACT_X."
   #endif
 
+  #if ENABLED(TOUCHMI_PROBE)
+    #if ((MIN_PROBE_EDGE == 0) || !defined(Z_SAFE_HOMING) || !defined(PROBE_RETRACT_HEIGHT))
```

Comment:
`MIN_PROBE_EDGE` should not be required. The option only exists for the benefit of inductive and capacitive probes that give unusual readings if they are too close to the edge of the bed. If the probe can reach the whole bed, you don't need `MIN_PROBE_EDGE`. And if you are using UBL, it doesn't care about the `MIN_PROBE_EDGE` either. If the probe can't reach the edge, UBL just assumes you will fill in the points as the next bed leveling step.

Reply:
Im not 100% sure about this one, its maybe to return on the bed "X" if the magnet is on the right and a few mm outside the bed. But you could remove the extra check

Reply:
Else, we have a second commit about the touchmi (tiny menu for manual Z probing)

Reply:
You are Right.
MIN_PROBE_EDGE not required.
It was required with boundaries and FIX_MOUNTED_PROBE. As this is a new implantation, it can be removed. Thanks